### PR TITLE
Document RAID section of partitioner

### PR DESCRIPTION
**Pull request by joseivanlopez**
PBI: https://trello.com/c/BQUekIB1/659-1-partitioner-specification-md-arrays.

### CODE REVIEW

File: doc/user-stories/expert-partitioner.md [hunk 1]
```diff
@@ -417,3 +420,143 @@ The specs are based on current version of expert partitioner for TW. The goal is
...
+  * and action 'Top' is selected
+    * reorder RAID devices by moving to top the current selected device from 'Selected Devices'
+  * and action 'Up' is selected
+    * reorder RAID devices by moving up the current selected device from 'Selected Devices'
+  * and action 'Down' is selected
+    * reorder RAID devices by moving down the current selected device from 'Selected Devices'
+  * and action 'Bottom' is selected
+    * reorder RAID devices by moving to bottom the current selected device from 'Selected Devices'
+  * and action 'Classify' is selected
+    * shows a dialog to classify the RAID devices
```
Comment (ancorgs) [inline doc/user-stories/expert-partitioner.md]: I guess we should have a line here saying that if you click next and the number of devices is below the minimum it shows an error pop-up and stays here.

File: doc/user-stories/expert-partitioner.md [hunk 2]
```diff
@@ -417,3 +420,143 @@ The specs are based on current version of expert partitioner for TW. The goal is
...
+    * and action 'Edit' is selected
+      * shows dialog to edit the RAID partition (similar to edit disk partition)
+    * and action 'Resize' is selected
+      * shows a dialog to modify the RADID devices
+    * and action 'Delete' is selected
+      * shows a confirm popup to delete the RAID
+      * deletes the RAID
+* and tab 'Used Devices' is selected
+  * shows a table with the columns: Device, Size, F, Enc and Type
+    * and the table is filled out with info of all devices that belong to the RAID
```
Comment (ancorgs) [inline doc/user-stories/expert-partitioner.md]: Isn't worth mentioning than selecting one item in the table can be used to "jump" to the corresponding partition?

File: doc/user-stories/expert-partitioner.md [hunk 3]
```diff
@@ -417,3 +420,143 @@ The specs are based on current version of expert partitioner for TW. The goal is
...
   * automatically appends /@ to the subvolume path
+
+### When 'RAID' is selected in the tree view
+* shows a table with the columns: Device, Size, F, Enc, Type, FS Type, Label, Mount Point, RAID Type, Chunk Size
+  * and the table is filled out with info of all RAID devices
+* *(pending) shows the actions: 'Add RAID', 'Edit', 'Resize' and 'Delete'*
+* *(pending) and action 'Add RAID' is selected*
+  * and there is less than two not mounted partitions
+    * shows an error message
+  * and there is at least two not mounted partitions with correct partition ID (not windows)
```
[the remark this answers is not in the document]
Comment (joseivanlopez) [reply doc/user-stories/expert-partitioner.md]: The allowed partition types seems to be the following ones: https://github.com/yast/yast-storage/blob/master/src/include/partitioning/raid_lib.rb#L65

File: doc/user-stories/expert-partitioner.md (hunk 2 above)
Comment (joseivanlopez) [reply doc/user-stories/expert-partitioner.md]: Ohhh, I got just realized about that :)

**Comment by ancorgs** [comment]
LEBTMN (looks even better to me now)

File: doc/user-stories/expert-partitioner.md [hunk 4]
```diff
@@ -417,3 +420,148 @@ The specs are based on current version of expert partitioner for TW. The goal is
...
+    * same wizard step than in partition creation (see above)
+
+#### *(pending) When we are in wizard step to select RAID type and devices*
+* shows the following options
+  * RAID Type
+    * RAID 0 (at least 2 devices)
+    * RAID 1 (at least 2 devices)
+    * RAID 5 (at least 3 devices)
+    * RAID 6 (at least 4 devices)
+    * RAID 10 (at least 2 devices)
```
Comment (jsrain) [inline doc/user-stories/expert-partitioner.md]: Don't we need at least 4 devices for RAID 10?

Comment (ancorgs) [reply doc/user-stories/expert-partitioner.md]: That's what I though when I saw that line. So I tested it and the expert partitioner allows you to continue with 2.

**Pull request merged**